public/js/visualize.js

```diff
@@ -1,55 +1,75 @@
 var urlParams = new URLSearchParams(window.location.search);
-var url = 'visualize/' + urlParams.getAll('articleID');
+var url = "visualize/" + urlParams.getAll("articleID");
 
 fetchGET(url, function(err, data) {
   if (err) {
     console.log(err);
```

Comment:
@Karyum Once we have an error page we could redirect the user to it in case of error but for now is it possible to keep the console.log until the error page is done? (I know it's not ideal..)